app/views/shared/_sidebar.html.erb

```diff
@@ -0,0 +1,6 @@
+<ul class='button-column'>
+  <li><%=link_to("Manage Groves", admin_groves_path, class: "btn btn-success btn-block") if policy(:grove).index? %></li>
```

Comment:
going forward, I think we will have two partials here: one for admin admin functionality, and one for teacher functionality.